### ISSUE TITLE
Unreadable iOS Status Bar with Night Theme

### ISSUE THREAD

**Issue by tatobox** [osmandapp/OsmAnd-iOS#3321]
### Description

When you switch to Night Theme status bar remain transparent and text/icon become white. This is not readable

### Steps to reproduce

Switch to Night Theme

### Actual result

Status Bar remain transparent and text/icon (date, time,battery, etc) become white

### Expected result

Status Bar become dark or text/icon become black

### Your Environment (required)

```
OsmAnd Version: Beta 4.6.2.8
Android/iOS version: iOS 15.7.2 and iPadOS 16.7.2
Device model: iPhone SE 1st gen and iPad Pro 12.9 1st gen
Crash-Logs: ?
```

**Comment by tatobox** [comment]
TestFlight 4.6.4.2
The issue is still here

**Comment by DmitryAlexei** [comment]
Still cannot reproduce. Please attach your profile settings (menu > settings > your profile > export) and say what theme do you have in the device's settings.

**Comment by dmpr0** [comment]
* Profile theme: Dark, Map: Light

<details>
    <summary>Screenshot</summary>
<table style="width:100%">
  <tr>
    <td><img src="https://github.com/osmandapp/OsmAnd-iOS/assets/9365308/8e1a929b-1a80-4035-b4e6-29e494b84e65"  width="300" height="auto"/> </td>
  </tr>
</table>  
</details>

**Comment by tatobox** [comment]
> Still cannot reproduce. Please attach your profile settings (menu > settings > your profile > export) and say what theme do you have in the device's settings.

Hi
my profile setting attached.
Device setting theme is Dark

[Guida.zip](https://github.com/osmandapp/OsmAnd-iOS/files/13767251/Guida.zip)

**Comment by dmpr0** [comment]
Done

<details>
    <summary>Screenshot</summary>
    <table style="width:100%">
      <tr>
        <th></th>
        <th></th>
      </tr>
      <tr>
        <td><img src="https://github.com/osmandapp/OsmAnd-iOS/assets/9365308/a5b19997-8d8a-476e-a844-ae414db10df2"  width="300" height="auto"/> </td>
        <td><img src="https://github.com/osmandapp/OsmAnd-iOS/assets/9365308/18bde624-7c4e-481b-ae19-3dba61c5982e"  width="300" height="auto"/> </td>
      </tr>
    </table>   
</details>   

Fix available in the latest TestFlight build 4.7.1.3, https://testflight.apple.com/join/7poGNCKy
